apps/exceptions.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+class UrlShortenerError(APIException):
+    status_code = status.HTTP_503_SERVICE_UNAVAILABLE
+    default_detail = 'Internal Service Error.'
```

Comment:
Internal service error is HTTP 500.

Reply:
This was just the same default_detail as the AWSIoTError and RPC error use. I can change it to be URL Shortener specific